src/rp2_common/hardware_i2c/include/hardware/i2c.h

```diff
@@ -155,6 +155,26 @@ static inline i2c_hw_t *i2c_get_hw(i2c_inst_t *i2c) {
  */
 int i2c_write_blocking_until(i2c_inst_t *i2c, uint8_t addr, const uint8_t *src, size_t len, bool nostop, absolute_time_t until);
 
+/*! \brief Attempt to write specified number of bytes to address, blocking until the specified absolute time is reached.
```

Comment:
Your `_mem` functions have the same `\brief` description (which is used by the [doxygen](https://raspberrypi.github.io/pico-sdk-doxygen/group__hardware__i2c.html)) as their non-mem counterparts. Might be nice if they were slightly different?

Reply:
You're absolutely correct, I've updated the `\brief`s now